modules/schedule_module/jsx/scheduleIndex.js

```diff
@@ -0,0 +1,592 @@
+      .then((resp) => resp.json())
+      .then((data) =>
+        this.setState({data}))
+      .then(()=>{
+         this.tabByid();
+       })
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+
+  fetchDataForm(type, value) {
+    return fetch(this.props.formURL+'/'+type+'/'+value, {credentials: 'same-origin'})
+      .then((resp) => resp.json())
+      .then((data) => {
+       if (type === 'DCCID' ) {
+      this.setState({formData: {...this.state.formData, PSCID: data['PSCID']}});
+      } else {
+      this.setState({formData: {...this.state.formData, DCCID: data['DCCID']}});
+      }
+      this.setState({formData: {...this.state.formData, SessionFieldOptions: data['Session']}});
+      })
+      .catch((error) => {
+        this.setState({error: true});
+        console.error(error);
+      });
+  }
+  /**
+   * Updates filter state
+   *
+   * @param {object} filter passed from FilterForm
+   */
+  updateFilter(filter) {
+    this.setState({filter});
+  }
+
+  /**
+   * Sets Filter to empty object
+   */
+  clearFilter() {
+    this.updateFilter({});
+    history.replaceState({}, '', '?');
+  }
+  /**
+   * Sets table data of tab by id
+   *
+   */
+  tabByid() {
+    const today = this.state.data.fieldOptions.today;
+    const next = this.state.data.fieldOptions.next30days;
+    const list = this.state.data.Data;
+    this.setState({tabledatapast: list.filter((e)=>e[7]<today)});
+    this.setState({tabledatanext: list.filter((e)=>{
+    return e[7]>today && e[7]<=next;
+    })});
+    this.setState({tabledatatoday: list.filter((e)=>e[7]==today)});
+  }
+  /**
+   * Store the value of the element in this.state.formData
+   *
+   * @param {string} formElement - name of the form element
+   * @param {string} value - value of the form element
+   */
+  setFormData(formElement, value) {
+    let formData = this.state.formData;
+    formData[formElement] = value;
+    if (formElement === 'DCCID') {
+    formData['PSCID'] = null;
+    this.fetchDataForm('DCCID', formData['DCCID']);
+    }
+    if (formElement === 'PSCID') {
+    formData['DCCID'] = null;
+    this.fetchDataForm('PSCID', formData['PSCID']);
+    }
+    this.setState({
+      formData: formData,
+    });
+  }
+  /**
+   * Handles the submission of the Add Schedule form
+   *
+   * @param {event} e - event of the form
+   */
+  handleSubmit(e) {
+    let formData = this.state.formData;
+    let formObject = new FormData();
+    for (let key in formData) {
+      if (formData[key] !== '') {
+        formObject.append(key, formData[key]);
+      }
+    }
+    formObject.append('edit', this.state.editModal);
+
+    fetch(this.props.submitURL, {
+      method: 'POST',
+      cache: 'no-cache',
+      credentials: 'same-origin',
+      body: formObject,
+    })
+    .then((resp) => {
+      if (resp.ok && resp.status === 200) {
+        swal.fire('Success!', 'Schedule modified.', 'success').then((result) => {
```

Comment:
you will need to dynamically display the swal text i think. unfortunately right now, the 'schedule modified' also shows when you create an appointment for the first time

<img width="878" alt="Screen Shot 2020-07-27 at 3 22 26 PM" src="https://user-images.githubusercontent.com/17415878/88582730-2a44c380-d01d-11ea-91d8-548d39558b56.png">